src/fsharp/CompileOps.fs

```diff
@@ -2433,7 +2445,7 @@ type TcConfig private (data : TcConfigBuilder,validate:bool) =
         let (_, fileNameOpt) as res = computeKnownDllReference(GetFSharpCoreLibraryName())
         match fileNameOpt with
         | None -> 
-            // if FSharp.Core was not provided explicitly - use version that was referenced by compiler
+            // if FSharp.Core was not provided explicitly - use version that was used by the compiler
```

Comment:
why we want to use fsc version of FSharp.Core instead of fail compilation if FSharp.Core is not passed as fsc argument?